### PR TITLE
[major mode] updated Clojure install and user guide

### PR DESCRIPTION
**Pull request by practicalli-johnny**
Install and user guide has been extracted from Practicalli Clojure and a new Practicalli VSpaceCode book has been created.

The install guide and user guide have been re-written and new screenshots taken.  Further content on developing with VSpaceCode will be added to this new book to enhance the user experience

https://practical.li/vspacecode/

### CODE REVIEW

**Comment by practicalli-johnny** [comment]
reduced the line by 12 characters to pass the 150 character lint rule

**Review by marcoieni — APPROVED** [review]

File: docs/major-mode.md [hunk 1]
```diff
@@ -45,8 +45,7 @@ Required extensions:
 
 Documentation:
 
-- [VSpaceCode and Calva install guide](http://practicalli.github.io/clojure/clojure-editors/editor-install-guides/vspacecode-calva.html#install-vs-code-extension)
-- [VSpaceCode and Calva user guide](http://practicalli.github.io/clojure/clojure-editors/editor-user-guides/vspacecode-calva.html)
+- [Practicalli VSpaceCode](https://practical.li/vspacecode/) install & guide for VSpaceCode and Calva for Clojure development (updated Febrary 2023)
```
Comment (stevenguh) [inline docs/major-mode.md]:
```suggestion
- [Practicalli VSpaceCode](https://practical.li/vspacecode/) install & guide for VSpaceCode and Calva for Clojure development
```

I would suggest removing the updated, but other than that LGTM. Thanks for updating

**Comment by stevenguh** [comment]
Just double checking, did you remove the repo by mistake?

**Comment by practicalli-johnny** [comment]
Ooops, I thought this had been merged.

Created #68 as previously forked repository was deleted

**Pull request closed without merging**